src/tests/ListCryptos.test.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { Provider, useSelector } from 'react-redux';
+import { MemoryRouter } from 'react-router-dom';
+import ListCryptos from '../components/ListCryptos';
+import store from '../redux/store';
+import '@testing-library/jest-dom/extend-expect';
+
+jest.mock('react-redux', () => ({
+  ...jest.requireActual('react-redux'),
+  useSelector: jest.fn(),
+}));
+
+describe('Home Component', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  test('renders the component', () => {
+    useSelector.mockReturnValue({ isloading: true, crytoList: [] });
+
+    render(
+      <Provider store={store}>
+        <MemoryRouter>
+          <ListCryptos />
+        </MemoryRouter>
+      </Provider>,
+    );
+    expect(
+      screen.getByPlaceholderText('Enter Currency Name'),
+    ).toBeInTheDocument();
+  });
+
+  test('should display coins', () => {
+    const mockCoins = [
+      { id: 1, symbol: 'BTC', percent_change_1h: 5 },
```

Comment:
- [ ] Please make sure that all your application test pass without fail to ensure that the application works without any issues. Currently, you have some of your tests failing so I suggest that you fix this :point_down: 
  
  ![image](https://github.com/DebyGrey/metric-app/assets/92303891/67926ee2-0022-4986-9b45-f9d63b6b6c04)